concent_api/conductor/tests/test_integration_verification.py

```diff
@@ -0,0 +1,276 @@
+from django.test    import override_settings
+from django.test    import TestCase
+from django.urls    import reverse
+
+from golem_messages import message
+
+from ..models       import UploadReport
+from ..models       import UploadRequest
+from ..models       import VerificationRequest
+from ..tasks        import verification_request_task
+
+
+class ConductorVerificationIntegrationTest(TestCase):
+
+    multi_db = True
+
+    def setUp(self):
+        self.file_path = 'blender/result/ef0dc1/ef0dc1.zzz523.zip'
+
+        self.compute_task_def = message.ComputeTaskDef()
+        self.compute_task_def['subtask_id'] = 'abc'
+
+    def test_conductor_should_return_404_when_file_path_parameter_not_matching_url_pattern_is_used(self):
+        response = self.client.get(
+            '/conductor/report-upload/blender/result/ef0dc1/ef0dc1.zzz523.arj',
+            content_type = 'application/octet-stream',
+        )
+
+        self.assertEqual(response.status_code, 404)
+
+    def test_conductor_should_create_upload_report(self):
+        response = self.client.get(
+            reverse(
+                'conductor:report_upload',
+                kwargs = {
+                    'file_path': self.file_path
+                }
+            ),
+            content_type = 'application/octet-stream',
+        )
+
+        self.assertEqual(response.status_code,         200)
+        self.assertEqual(UploadReport.objects.count(), 1)
+
+        upload_report = UploadReport.objects.first()
+        self.assertEqual(upload_report.path,           self.file_path)
+        self.assertEqual(upload_report.upload_request, None)
+
+    def test_conductor_should_create_upload_report_and_link_to_related_upload_request(self):
+        verification_request = VerificationRequest(
```

Comment:
please extract these few lines to `_prepare_verification_request` or sth similar; in other tests too

Reply:
done :+1: